recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
     - patches/0001-unconditional-geos-pc.patch
 
 build:
-  number: 0
+  number: 1
```

Comment:
B/c this would have passed in build 0, we don't really a new package just b/c of the test change. Let's go back to 0 and avoid an unnecessary upload.

Reply:
Good point. This is also a very low priority.